filament/src/PostProcessManager.cpp

```diff
@@ -183,4 +185,176 @@ void PostProcessManager::finish(driver::TargetBufferFlags discarded,
     commands.clear();
 }
 
+
+// ------------------------------------------------------------------------------------------------
+
+FrameGraphResource PostProcessManager::msaa(FrameGraph& fg,
+        FrameGraphResource input, driver::TextureFormat outFormat) noexcept {
+
+    struct PostProcessMSAA {
+        FrameGraphResource input;
+        FrameGraphResource output;
+    };
+
+    auto& ppMSAA = fg.addPass<PostProcessMSAA>("msaa",
+            [&](FrameGraph::Builder& builder, PostProcessMSAA& data) {
+                auto const* inputDesc = fg.getDescriptor(input);
+                data.input = builder.read(input);
+
+                FrameGraphResource::Descriptor outputDesc{
+                        .width = inputDesc->width,
+                        .height = inputDesc->height,
+                        .format = outFormat
+                };
+                data.output = builder.write(builder.createResource("msaa output", outputDesc));
+            },
+            [=](
```

Comment:
I think this can be `[]` instead of `[=]` since you aren't using any captures in this particular lambda.